### ISSUE TITLE
Add a `Once` option to display some text only one time

### ISSUE THREAD

**Issue by lizadaly** [lizadaly/windrift#234]
This is more likely to apply to multiplayer games but can be general-purpose enough to apply to any story, so it should probably go in `core`.

Normally Windrift is stateless and will display a response just based on the current state of the world, without regard to history.

Without excessively complicating the API, it should be possible to design text to displayed only once. Ideally this would be added to `Response` as a flag and otherwise just follow the usual convention.

This could be implemented in a few ways:

* Use the `log` to check whether the choice has been made before

Might be fragile. So far I've avoided using the `log` for anything beyond event synchronization in multiplayer, and an optional component for debugging. I think ideally the log is mostly considered a private component and not something an author needs to understand.

* Use a second implicit choice/tag to track whether it has been displayed before.

I don't like this as the inventory is already cluttered with nav items. It makes it harder for the author to debug story state if the inventory/choice store is overwhelmed.

* Use a browser cookie/local storage variable directly, bypassing the inventory mechanism.

This adds a new storage mechanism, and thus something else that needs to be deleted when resetting a story.

* Use React local state.

This might be the best choice from an implementation perspective as it's straightforward to code, even though it's the most likely to produce duplicate results: if the user refreshes their browser they'll get this text again. Depending on how this is authored it might produce very inconsistent results (like a message about a player that's no longer in the room). 

On the other hand, users won't do that often, and I don't know how frequently this feature will be used by authors. It might be best to start with this, document that it's not guaranteed to not duplicate, and go from there. Authors can probably hack together something with other checks to avoid highly confusing output.

**Comment by lizadaly** [comment]
Now that multiplayer games are always rehydrated from the remote db, using React state isn't reliable. 

What does "only once" really mean? Once per chapter? I think this breaks the stateless model too much. Closing for now.
